libs/sdk-ui-dashboard/src/presentation/filterBar/tests/utils.test.ts

```diff
@@ -1,7 +1,12 @@
 // (C) 2023 GoodData Corporation
 import { describe, it, expect, vi } from "vitest";
-import { getVisibilityIcon } from "../utils.js";
-import { DashboardDateFilterConfigModeValues } from "@gooddata/sdk-model";
+import { areAllFiltersHidden, getVisibilityIcon } from "../utils.js";
+import {
```

Comment:
Can you please re-arrange the import to make it more easy to read? Please move lines 4 - 8 below line 2, and add an empty line to separate the import library and import files in the same project.
Ex:
```javascript
import { describe, it, expect, vi } from "vitest";
import {
DashboardAttributeFilterConfigModeValues,
    DashboardDateFilterConfigModeValues,
} from "@gooddata/sdk-model";
import cloneDeep from "lodash/cloneDeep.js";

import { areAllFiltersHidden, getVisibilityIcon } from "../utils.js";
import { filterBarAttributeFilterIndexes } from "./filterBar.fixture.js";
```

Reply:
updated!

Reply:
Are there any rules for this?

Reply:
Anyway, updated!